cmd/rocsp-tool/client_test.go

```diff
@@ -111,3 +114,54 @@ func TestStoreResponse(t *testing.T) {
 	err = cl.storeResponse(context.Background(), response, &ttl)
 	test.AssertNotError(t, err, "storing response")
 }
+
+type mockOCSPGenerator struct{}
+
+func (mog mockOCSPGenerator) GenerateOCSP(ctx context.Context, in *capb.GenerateOCSPRequest, opts ...grpc.CallOption) (*capb.OCSPResponse, error) {
+	return &capb.OCSPResponse{
+		Response: []byte("phthpbt"),
+	}, nil
+
+}
+
+func TestLoadFromDB(t *testing.T) {
+	redisClient, clk := makeClient()
+
+	dbMap, err := sa.NewDbMap(vars.DBConnSA, sa.DbSettings{})
+	if err != nil {
+		t.Fatalf("Failed to create dbMap: %s", err)
```

Comment:
Any reason you're using
```
if err != nil {
   t.Fatalf(...)
}
```
instead of
```
test.AssertNotError(t, err, ...)
```
?

Reply:
Mainly that I copy-pasted from somewhere else that used the same pattern. :-)